src/gui/connectionvalidator.h

```diff
@@ -97,7 +97,7 @@ class ConnectionValidator : public QObject
     Q_ENUM(Status);
 
     // How often should the Application ask this object to check for the connection?
-    enum { DefaultCallingIntervalMsec = 32 * 1000 };
+    enum { DefaultCallingIntervalMsec = 62 * 1000 };
```

Comment:
How does a change from 32 to 62 implement a 57 seconds timeout?
Some hidden magic eating 5secs?

Reply:
Yes, there's some magic that makes the timeout 5s less than the calling interval.